libs/ojsonnet/Eval_jsonnet_subst.ml

```diff
@@ -1,13 +1,24 @@
 open Core_jsonnet
 open Common
-module V = Value_jsonnet_subst
+module V = Value_jsonnet
 module A = AST_jsonnet
 module J = JSON
 
 exception Error of string * Tok.t
 
 type cmp = Inf | Eq | Sup
 
+(*Creates std so that we can add it to the environment when we switch back to
+  * environment model for handling standard library functions
+*)
+let pre_std = Std_jsonnet.get_std_jsonnet ()
+
+let path =
+  match Fpath.of_string "../" with
```

Comment:
I don't understand this code. What is path? Seems like it's aways .. so maybe the variable
should be renamed to let parent ?

Reply:
The filepath is a placeholder (we don't actually care about what it is, since it's not referenced, just there to make the types work out). I can add a comment explaining that!